drupal/islandora/include/admin.form.inc

```diff
@@ -59,3 +67,26 @@ function islandora_config_form_validate_stomp_url(array $element, array &$form_s
     form_error($element, t('Connection failed for @url: @message', array('@url' => $url, '@message' => $e->getMessage())));
   }
 }
+
+/**
+ * Karaf url validator.
+ *
+ * @param array $element
+ *   Form element for stomp broker url.
+ * @param array $form_state
+ *   Drupal form state.
+ * @param array $form
+ *   Drupal renderable array.
+ */
+function islandora_config_form_validate_karaf_url(array $element, array &$form_state, array $form) {
+  // Try to connect to the server.
+  $url = $element['#value'] + '/cxf';
+  $handle = curl_init($url);
+  curl_setopt($handle,  CURLOPT_RETURNTRANSFER, TRUE);
+  $response = curl_exec($handle);
+  $httpCode = curl_getinfo($handle, CURLINFO_HTTP_CODE);
+  if($httpCode != 200) {
```

Comment:
Travis error - could you add a space after the ```if```, also change the ```$httpCode``` to match the change above.